h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java

```diff
@@ -3,11 +3,11 @@
 import hex.Model;
 import hex.MultiModelMojoWriter;
 import hex.StackedEnsembleModel;
+import water.DKV;
```

Comment:
Why this import?

Reply:
Not sure, I did not touch that part.

Reply:
Or rather, why was it added if it is already there?

Reply:
Wait, it is there in master originally.  IntelliJ changed its location.

Reply:
Yep, agree @michalkurka